src/components/settings-drawer.tsx

```diff
@@ -21,20 +21,54 @@ export interface SettingsDrawerProps extends DrawerProps {
 export default function SettingsDrawer(props: SettingsDrawerProps): React.ReactElement {
   const classes = useStyles();
   const { settings, onSettingsChange, ...otherProps } = props;
-  const { trajectoryAnimation } = settings;
+  const { trajectoryAnimation, trajectoryDiameter, trajectoryColor } = settings;
 
   const trajAnimsText = React.useMemo(
     () => Object.keys(TrajectoryAnimation).slice(Object.keys(TrajectoryAnimation).length * 0.5),
     [],
   );
 
-  const drawerAnchor = useMediaQuery('(max-aspect-ratio: 8/10)') ? 'bottom' : 'right';
+  const trajDiameterText = React.useMemo(() => {
+    const keys = Object.keys(TrajectoryDiameter).slice(
+      Object.keys(TrajectoryDiameter).length * 0.5,
+    );
+    return keys.map(key => {
+      let text = '';
+      key.split(/(?=[A-Z])/).forEach(k => {
+        text += k + ' ';
+      });
+      return text.trim();
+    });
```

Comment:
Since `TrajectoryDiameter` is a enum, you can just "hardcode" the text in an object.

https://www.typescriptlang.org/play?#code/KYOwrgtgBAKgTgQwFbAMYBcD2cCeARASwQmHWDigG8AoKKAMQIA8BlAgL2ABpaoAlTACNM6Npx4BfatVSYQAZ3RR0iFBmz4iJMnBjAm6AFxUoAbQDWUAiFiq0WXIWKlyAXWOK41gOZQJUAF4qXlN4ZHsNJ21yADpGVg5gdygAcnjgABMoMWAUnjpQu3VHLRc4GIFhUUTklMqRbMS86gkAbmlZBUwAG2AY7sxvAAoVcOLNZx09A0Kxhwno8vic1wBKds75Hr6B4dG1eaiy6fRZg8jSnQqhERX16iA